lib/core/constants/preferences_flags.dart

```diff
@@ -10,4 +9,7 @@ enum PreferencesFlag {
 
   // Theme flag
   theme,
+
+  // Welcome flag
+  welcome,
```

Comment:
Why? I'm not sure to understand

Reply:
Yes I don't think it's necessary anymore, I'll remove it (I added it when I didn't know the lib were using their own shared-pref behind the scene)